pkgs/compute_statistics/bin/compute_statistics.dart

```diff
@@ -0,0 +1,45 @@
+import 'dart:io';
+
+import 'package:args/args.dart';
+import 'package:compute_statistics/compute_statistics.dart';
+import 'package:compute_statistics/statistics_type.dart';
+import 'package:dart_triage_updater/firebase_database.dart';
+
+Future<void> main(List<String> arguments) async {
```

Comment:
I think its the same either way, but this functionality could also be added to the existing cli tool in this repo as a separate command (convert that cli tool to supporting multiple commands).

Reply:
It feels a bit cleaner this way, but merging the two does make sense.